src/components/Moviepage/MoviePage.jsx

```diff
@@ -1,5 +1,69 @@
-import React from "react";
+import axios from "axios";
+import React, { useEffect, useState } from "react";
+import { useParams } from "react-router-dom";
+import Spinner from "../Spinner/Spinner";
 
-export default function MoviePage({ movieInfo }) {
-  return <div>{movieInfo.id}</div>;
+const api_key = process.env.REACT_APP_API_KEY;
+
+export default function MoviePage({ isLoading, setIsLoading }) {
```

Comment:
isLoading and setIsLoading should be private to that component

Reply:
fixed